incubator/hnc/third_party/open-policy-agent/gatekeeper/pkg/webhook/certs.go

```diff
@@ -517,13 +566,59 @@ func (r *ReconcileVWH) Reconcile(request reconcile.Request) (reconcile.Result, e
 		if err := r.client.Update(r.ctx, vwh); err != nil {
 			return reconcile.Result{Requeue: true}, err
 		}
+
+		// Ensure certs on CRD conversion webhooks if there's any.
+		if result, err := r.ensureCRDConvWHCerts(artifacts.CertPEM); err != nil {
+			return result, err
+		}
+
+		if !r.wasCAInjected.Load() {
+			r.wasCAInjected.Store(true)
+		}
+	}
+
+	return reconcile.Result{}, nil
+}
+
+func (r *ReconcileWH) ensureCRDConvWHCerts(certPem []byte) (reconcile.Result, error) {
```

Comment:
Updated the logic for returns here and commented in https://github.com/kubernetes-sigs/multi-tenancy/pull/953#discussion_r463172373 .